package.json

```diff
@@ -30,6 +30,7 @@
     "@babel/eslint-parser": "~7.21.3",
     "@babel/plugin-syntax-import-assertions": "~7.20.0",
     "@desertnet/html-parser": "~1.0.1",
+    "@google-cloud/logging-winston": "5.3.0",
```

Comment:
This dependency in mdn-bcd-collector was really there to make logging in GCP work. For a script run locally, here in BCD, I think we should use the logging from test/utils.ts.